fe/fe-core/src/main/java/org/apache/doris/qe/QeProcessorImpl.java

```diff
@@ -105,6 +106,26 @@ public void registerQuery(TUniqueId queryId, QueryInfo info) throws UserExceptio
         }
     }
 
+    @Override
+    public QueryInfo getQueryInfoByQueryId(TUniqueId queryId) throws UserException {
```

Comment:
Both query start time and token are in coordinator, we could get these info directly from coordinator.
And QueryInfo contains a coordinator. So that we do not need these API any more.